TLM/TLM/Resources/lang.txt

```diff
@@ -230,8 +230,9 @@ Vehicles_may_turn_on_red Vehicles may turn at red traffic lights
 Also_apply_to_left/right_turns_between_one-way_streets Also apply to left & right turns between one-way streets
 Scan_for_known_incompatible_mods_on_startup Scan for known incompatible mods on startup
 Ignore_disabled_mods Ignore disabled mods
-Traffic_Manager_detected_incompatible_mods Traffic Manager detected incompatible mods
+Traffic_Manager_detected_incompatible_mods detected incompatible mods
```

Comment:
Capitalize D in Detected

Reply:
Uhm, that would look weird - it would make Detected look like part of the mod name.

Reply:
I guess I misunderstood where this was used.